

### **Problem Statement:**

A home-built model rocket is launched vertically upward. The rocket’s position as a function of time $t$ (in seconds) during the fuel burn phase $0 < t < t_0$ is given by:

$$
y(t) = \frac{1}{2} (a_0 - g) t^2 - \frac{1}{30} a_0 \frac{t^6}{t_0^4}
$$

where:

* $a_0$ is a positive constant acceleration due to the rocket engine, with $a_0 > g$,
* $g$ is the acceleration due to gravity ($9.8 \, m/s^2$),
* $t_0$ is the burn-out time of the rocket’s engine.

---

### **Question:**

Find the y-component of the acceleration $a_y(t)$ of the rocket as a function of time $t$ for $0 < t < t_0$.

Express your answer in terms of $a_0$, $g$, $t$, and $t_0$.



Got it! You want to find the **y-component of acceleration $a_y(t)$** from the given position function:

$$
y(t) = \frac{1}{2}(a_0 - g) t^2 - \frac{1}{30} a_0 \frac{t^6}{t_0^4}, \quad 0 < t < t_0
$$

where

* $a_0$ is a positive constant with $a_0 > g$,
* $g$ is gravitational acceleration,
* $t_0$ is the fuel burn-out time.

---

### Step-by-step solution:

---

**Step 1: Recall acceleration as second derivative of position:**

$$
a_y(t) = \frac{d^2 y}{dt^2}
$$

---

**Step 2: Differentiate $y(t)$ once to get velocity $v_y(t)$:**

$$
v_y(t) = \frac{dy}{dt} = \frac{d}{dt} \left[\frac{1}{2}(a_0 - g) t^2\right] - \frac{d}{dt} \left[\frac{1}{30} a_0 \frac{t^6}{t_0^4}\right]
$$

* First term:

$$
\frac{d}{dt} \left[\frac{1}{2}(a_0 - g) t^2\right] = (a_0 - g) t
$$

* Second term:

$$
\frac{d}{dt} \left[\frac{1}{30} a_0 \frac{t^6}{t_0^4}\right] = \frac{1}{30} a_0 \frac{6 t^5}{t_0^4} = \frac{1}{5} a_0 \frac{t^5}{t_0^4}
$$

So,

$$
v_y(t) = (a_0 - g) t - \frac{1}{5} a_0 \frac{t^5}{t_0^4}
$$

---

**Step 3: Differentiate $v_y(t)$ to get acceleration $a_y(t)$:**

$$
a_y(t) = \frac{dv_y}{dt} = \frac{d}{dt} \left[(a_0 - g) t\right] - \frac{d}{dt} \left[\frac{1}{5} a_0 \frac{t^5}{t_0^4}\right]
$$

* First term derivative:

$$
\frac{d}{dt} \left[(a_0 - g) t\right] = (a_0 - g)
$$

* Second term derivative:

$$
\frac{d}{dt} \left[\frac{1}{5} a_0 \frac{t^5}{t_0^4}\right] = \frac{1}{5} a_0 \frac{5 t^4}{t_0^4} = a_0 \frac{t^4}{t_0^4}
$$

So,

$$
a_y(t) = (a_0 - g) - a_0 \frac{t^4}{t_0^4}
$$

---

### **Final answer:**

$$
\boxed{
a_y(t) = (a_0 - g) - a_0 \left(\frac{t}{t_0}\right)^4, \quad 0 < t < t_0
}
$$

---

### **Interpretation:**

* At $t=0$, acceleration is $a_y(0) = (a_0 - g)$.
* At $t = t_0$, acceleration is

$$
a_y(t_0) = (a_0 - g) - a_0 = -g
$$

which matches the expected acceleration after fuel burnout (only gravity acting).

---

If you want, I can provide the Python code to symbolically compute this using `sympy`. Let me know!


In [1]:
import sympy as sp

# Define symbols
t, a0, g, t0 = sp.symbols('t a0 g t0', real=True, positive=True)

# Position function y(t)
y = (1/2)*(a0 - g)*t**2 - (1/30)*a0*(t**6)/(t0**4)

# Velocity v(t) = dy/dt
v = sp.diff(y, t)

# Acceleration a(t) = d^2y/dt^2
a = sp.diff(v, t)

# Simplify the acceleration expression
a_simplified = sp.simplify(a)

print("Position y(t):")
sp.pprint(y)

print("\nVelocity v(t) = dy/dt:")
sp.pprint(v)

print("\nAcceleration a(t) = d^2y/dt^2:")
sp.pprint(a_simplified)


Position y(t):
                         6                      
  0.0333333333333333⋅a₀⋅t     2                 
- ──────────────────────── + t ⋅(0.5⋅a₀ - 0.5⋅g)
              4                                 
            t₀                                  

Velocity v(t) = dy/dt:
          5                       
  0.2⋅a₀⋅t                        
- ───────── + 2⋅t⋅(0.5⋅a₀ - 0.5⋅g)
       4                          
     t₀                           

Acceleration a(t) = d^2y/dt^2:
          4                 
  1.0⋅a₀⋅t                  
- ───────── + 1.0⋅a₀ - 1.0⋅g
       4                    
     t₀                     


In [2]:
import sympy as sp

# Define symbols
t, a0, g, t0 = sp.symbols('t a0 g t0', real=True, positive=True)

# Position function y(t)
y = (1/2)*(a0 - g)*t**2 - (1/30)*a0*(t**6)/(t0**4)

# Velocity v(t) = dy/dt
v = sp.diff(y, t)

# Acceleration a(t) = d^2y/dt^2
a = sp.diff(v, t)

# Simplify the acceleration expression
a_simplified = sp.simplify(a)

# Printing expressions with latex formatting (for clarity)
print("Position y(t):")
print(sp.latex(y))

print("\nVelocity v(t) = dy/dt:")
print(sp.latex(v))

print("\nAcceleration a(t) = d^2y/dt^2:")
print(sp.latex(a_simplified))


Position y(t):
- \frac{0.0333333333333333 a_{0} t^{6}}{t_{0}^{4}} + t^{2} \left(0.5 a_{0} - 0.5 g\right)

Velocity v(t) = dy/dt:
- \frac{0.2 a_{0} t^{5}}{t_{0}^{4}} + 2 t \left(0.5 a_{0} - 0.5 g\right)

Acceleration a(t) = d^2y/dt^2:
- \frac{1.0 a_{0} t^{4}}{t_{0}^{4}} + 1.0 a_{0} - 1.0 g


Position y(t):
- $$\frac{0.0333333333333333 a_{0} t^{6}}{t_{0}^{4}} + t^{2} \left(0.5 a_{0} - 0.5 g\right)$$

Velocity v(t) = dy/dt:
- $$\frac{0.2 a_{0} t^{5}}{t_{0}^{4}} + 2 t \left(0.5 a_{0} - 0.5 g\right)$$

Acceleration a(t) = d^2y/dt^2:
- $$\frac{1.0 a_{0} t^{4}}{t_{0}^{4}} + 1.0 a_{0} - 1.0 g$$